README-incubator.md

```diff
@@ -0,0 +1,166 @@
+# JavaFX Incubator Modules
+
+## Overview
+
+This document includes instructions for adding a JavaFX incubator module to the JavaFX build.
+
+## Adding an Incubator module
+
+To add an incubator module, do the following:
+
+1. Add an entry for your module in `settings-incubator.gradle` and `build-incubator.gradle` in the designated place
+2. Create your module under `modules/jfx.incubator.YOURMODULENAME`, including your source code and test code as is done for other modules
+3. Add the needed build logic in `modules/jfx.incubator.YOURMODULENAME/project.gradle`
+
+Here is an example patch:
+
+```
+diff --git a/incubator-build.gradle b/incubator-build.gradle
```

Comment:
the diffs are rather hard to read.

maybe instead we should provide a clean example, with the module name token in UPPERCASE that will be easy to replace.

the idea is to give the author of an incubator module a template that they can easily use, rather than forcing them to decipher the diffs.

example:

```
ext.incubatorProjectNames = [
     'incubator.INCUBATORMODULE'
]
```



what do you think?